src/extensions.rs

```diff
@@ -8,8 +8,10 @@ use crate::shell::MessageInfo;
 pub const STRIPPED_BINS: &[&str] = &[crate::docker::DOCKER, crate::docker::PODMAN, "cargo"];
 
 pub trait CommandExt {
+    #[track_caller]
     fn fmt_message(&self, msg_info: &mut MessageInfo) -> String;
 
```

Comment:
I don't think this `#[track_caller]` is needed

Reply:
It's can't fail so I'd agree.

Reply:
It's not that it cant fail, it's that the `#[track_caller]` is what makes `std::panic::Location` return a correct value, but since all calls to `fmt_message` are behind another track_caller, we should be fine